pandas_transform/pandas_transform.md

```diff
@@ -0,0 +1,995 @@
+<!--
+
+author:   Elizabeth Drellich
+email:    drelliche@chop.edu
+version:  0.0.1
+module_template_version: 2.0.0
+language: en
+narrator: UK English Female
+title: Using `pandas` for Tabular Data in python
+comment:  This is an introduction to tabular data in python using `pandas` DataFrames.
```

Comment:
Title and comment now strongly emphasize "tabular", but that's not actually directly addressed in the module content --- although of course all of the examples are on tabular data, there's no discussion around that. I think "Transform data with pandas" or something along those lines is a clearer indication of what the module actually teaches.